src/pages/AppDetail.svelte

```diff
@@ -3,17 +3,45 @@
   import Pill from "../components/Pill.svelte";
   import { fetchJSON } from "../state/api";
   import FilterInput from "../components/FilterInput.svelte";
+  import Pagination, {
+    makePages,
+    goToPage,
+  } from "../components/Pagination.svelte";
 
   export let params;
   const URL = `data/${params.app}/index.json`;
   let app;
+  let to;
+  let from;
+  let total;
+  let metrics = [];
+  let pages = [];
+  let paginationData;
+  let lastPage;
+  let perPage = 100;
```

Comment:
Since this never varies, I think we can leave it out. Can you make 100 the default in the Pagination component?

Reply:
Sure, noted.